.travis.yml

```
@@ -26,9 +26,14 @@ env:
 # This should only fail with pip 10.0.1 with URL requirements (e.g., "package @ URL")
 # Useful for testing un-released upstream fixes
 matrix:
+  include:
+  - python: "3.8-dev"
+    env: INSTALL_TYPE="develop"
```

If you're going to make this `allow_failure`, what about setting

```suggestion
    env: INSTALL_TYPE="develop" PYTHONWARNINGS="error"
```

So we can make upcoming deprecations more visible?